README.md

```diff
@@ -235,6 +235,13 @@ StrimziKafkaCluster kafkaCluster = new StrimziKafkaCluster.StrimziKafkaClusterBu
 kafkaCluster.start();
 ```
 
+#### x) Logging Kafka Container/Cluster Output to SLF4J
```

Comment:
The text is good. But it looks like you have `ix`, `xi` and then `x`?

Reply:
updated

Reply:
Yeah, but you are still missing `x` -> the previous one should be `x` and this one `xi`?